### PR TITLE
PLT-3201 Code Theme Saves Upon Update

### PR DESCRIPTION
**Pull request by DavidLu1997**
Previously custom code themes did not save. Now they do save!

### CODE REVIEW

**Comment by coreyhulen** [comment]
+1

**Comment by jwilander** [comment]
+1

**Comment by enahum** [comment]
+1 defering to @jwilander

File: webapp/components/user_settings/custom_theme_chooser.jsx [hunk 1]
```diff
@@ -108,15 +108,24 @@ class CustomThemeChooser extends React.Component {
...
         $('.color-picker').colorpicker({
             format: 'hex'
         });
         $('.color-picker').on('changeColor', this.onPickerChange);
+        $('.group--code').on('change', this.onCodeThemeChange);
+    }
+
+    componentWillUnmount() {
+        $('.color-picker').off('changeColor', this.onPickerChange);
+        $('.group--code').off('changeColor', this.onCodeThemeChange);
```
Comment (jwilander) [inline webapp/components/user_settings/custom_theme_chooser.jsx]: I think this should be
`$('.group--code').off('change', this.onCodeThemeChange);`
right?

**Comment by DavidLu1997** [comment]
@jwilander `changeColor` fixed

**Comment by jwilander** [comment]
+1

**Pull request merged**